app/views/manage_assignments/outcome_coverages/_outcome_coverage.html.erb

```diff
@@ -37,7 +37,10 @@
     <div class="class-card-assignment-controls">
       <% if outcome_coverage.assignment.present? %>
         <% if outcome_coverage.assignment.attachments.present? %>
-          <%= link_to t(".attachments-expandable-link"), "#", class: "attachments-expandable-link" %>
+          <%= link_to "#", class: "attachments-expandable-link" do %>
+            <%= t(".attachments-expandable-link") %>
+            <span><%= outcome_coverage.assignment.attachments.count %></span>
```

Comment:
Calling `.count` on an association will *always* trigger a database query to get a count of records. This can become a performance problem.

Calling `.length` will always count the number of items loaded into the array in memory. This is performant, but can be the wrong thing to do if all you really need is the count of records (i.e. you do not need to get the actual attachments, just the count).

There's a third option, `.size` which will act like `length` if the association is already loaded but act like `count` if it's not. That's *generally* what I'd use in these situations, but you could make an argument for `length` here instead.